js/misc/paygManager.js

```diff
@@ -110,8 +129,11 @@ var PaygManager = new Lang.Class({
         this.emit('code-expired');
     },
 
-    addCode: function(code) {
-        this._proxy.AddCodeRemote(code);
+    addCode: function(code, callback) {
+        this._proxy.AddCodeRemote(code, (result, proxyError) => {
+            if (callback)
+                callback(proxyToPaygError(proxyError));
```

Comment:
`proxyToPaygError()` doesn’t seem to be defined anywhere.

Reply:
Oops! Somehow I messed up right before pushing my branch, since this belongs to a very old version that I no longer use. I guess I didn't notice because I was using a custom implementation of the manager for development, and the real thing (integration with the D-Bus daemon is not really tested). 

Will fix that, of course